events-api/src/modules/events/dto/requests/list-events-params.dto.ts

```diff
@@ -45,6 +45,16 @@ export class ListEventsParamsReqDto {
   @Max(1000)
   limit?: number;
 
+  @IsOptional()
+  @IsString()
+  @IsIn(['eventName', 'eventDate', 'eventCreatedAt', 'eventUpdatedAt'])
+  sort_by?: string;
```

Comment:
I'd suggest to keep the possible values of  sort by in some kind of constant. Then you can have better types here too

Reply:
fixed